boards/imix/src/main.rs

```diff
@@ -143,6 +145,10 @@ struct Imix {
     >,
     nrf51822: &'static capsules::nrf51822_serialization::Nrf51822Serialization<'static>,
     nonvolatile_storage: &'static capsules::nonvolatile_storage_driver::NonvolatileStorage<'static>,
+    printlog: &'static capsules::print_log::PrintLog<
+        'static,
+        VirtualMuxAlarm<'static, sam4l::ast::Ast<'static>>,
+    >,
```

Comment:
So now Imix exports both console and printlog .. is there a reason that a userspace app would want to use both at the same time?

Reply:
They have different semantics. Console always fully prints the userspace message (barring failures), while printlog will truncate if it's too long.

printlog also doesn't handle reading from the console.

I'm not arguing that imix should definitely always have both going forward, but they are different.

Reply:
Marking as resolved, this seems like the same discussion as in my other comment